NewPass/HapticEngine.swift

```diff
@@ -9,13 +9,15 @@
 import UIKit
 
 class HapticEngine {
-    
-    static func hapticWarning() {
+
+    static let shared = HapticEngine()
```

Comment:
Just a small suggestion. I know if seems really clean when you later do this: 

```swift
HapticEngine.shared.hapticTap(impactStyle: .light)
``` 

Or when you do this before this PR: 

```swift
HapticEngine.hapticTap(impactStyle: .light)
```

But I would recommend making this not static either. (Like you did with the other change in this PR.) 

so basically, try removing line 13 here, and when you need this just do something like this: 
```swift
HapticEngine().hapticTap(impactStyle: .light)
```
or this: 
```swift
let hapticEngine = HapticEngine()
hapticEngine.hapticTap(impactStyle: .light)
```

The reason is that static variables or objects are generally kept alive in memory for the lifetime of your application. Whereas in my example, you can create the HapticEngine right before you need it, then after you use it, the system will de-allocate it from memory. It doesn't really matter here because we are talking about very little memory anyway, but in theory it is better to avoid static types unless you actually need it to be static for some other reason. Does that make sense? 

Anyway, just a suggestion... Overall this PR is great!! 🎉

Reply:
Ah, I gotcha. That's good feedback. Thanks. I'll add that in.

Reply:
[Here's the changes](https://github.com/slo-addy/NewPass/pull/7). Let me know how that looks.